plugin-server/src/worker/ingestion/person-state.ts

```diff
@@ -778,7 +785,7 @@ export class DeferredPersonOverrideWriter {
 
             const messages: ProducerRecord[] = []
             for (const { id, ...mergeOperation } of rows) {
-                messages.push(...(await writer.addPersonOverride(tx, mergeOperation)))
+                messages.push(...(await this.writer.addPersonOverride(tx, mergeOperation)))
```

Comment:
Something important I hadn't considered before: Team on the override model is a foreign key (although it's _not_ one on the mapping table, just a bigint). I hadn't defined it as a foreign key on the pending override table, so this is either going to need to gracefully handle foreign key constraint violations, or those schemas are going to need to be brought into alignment. We'll also need to decide what to use on the new table. My initial assumption is that its probably fine to use a non-foreign key on the new table, worst case is we end up storing some extra overrides for longer than needed. Plus we're not propagating cascading deletes to ClickHouse anyway.